src/main/java/jewellery/inventory/service/OrganizationService.java

```diff
@@ -0,0 +1,68 @@
+package jewellery.inventory.service;
+
+import java.util.*;
+
+import jewellery.inventory.aspect.annotation.LogCreateEvent;
+import jewellery.inventory.dto.request.OrganizationRequestDto;
+import jewellery.inventory.dto.response.OrganizationResponseDto;
+import jewellery.inventory.exception.not_found.OrganizationNotFoundException;
+import jewellery.inventory.mapper.OrganizationMapper;
+import jewellery.inventory.model.*;
+import jewellery.inventory.repository.*;
+import jewellery.inventory.service.security.AuthService;
+import lombok.AllArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class OrganizationService {
+  private static final Logger logger = LogManager.getLogger(OrganizationService.class);
+  private final OrganizationRepository organizationRepository;
+  private final OrganizationMapper organizationMapper;
+  private final AuthService authService;
+  private final UserService userService;
+
+  private List<Organization> getAll() {
+    logger.debug("Fetching all organizations");
+    return organizationRepository.findAll();
+  }
+
+  private Organization getOrganization(UUID id) {
+    logger.info("Get organization by ID: {}", id);
```

Comment:
What was our strategy for logging? I think fetching data should be debug and altering data in db should be info. Also, I see we are using this method in `getOrganizationResponse` and we are loggin the same there. Perhaps remove it from here and leave it only there?